cmd/server-manager/plugins/events.lua

```diff
@@ -1,9 +1,13 @@
 json = require "json"
 utils = require "utils"
 
+-- Insert json folder path here
+local jsonPath = "your_plugin_folder"
+local contentPath = "your_content_path"
+
 -- these are lua hooks related to the manager itself, for help please view lua_readme.md!
 -- there are some example functions here to give you an idea of what is possible, feel free to write your own!
--- if you do and think other people would be interested in them consider making a pull request at https://github.com/JustaPenguin/assetto-server-manager
+-- if you do and think other people would be interested in them consider making a pull request at https://github.com/cj123/assetto-server-manager
```

Comment:
``https://github.com/JustaPenguin/assetto-server-manager`` is the right URL, cj123 just redirects to there